R/path.R

```diff
@@ -21,6 +21,9 @@ this_file_name <- function() {
 #' @md
 #' @export
 this_file_path <- function() {
+  if(!requireNamespace("this.path", quietly = TRUE)) {
+    abort("The package \"this.path\" is required.")  
+  }
   stopifnot(requireNamespace("this.path", quietly = TRUE))
```

Comment:
I dropped this given it's covered by the new check above.

5d23ccd (this_file_path: drop redundant guard, 2025-06-09)